lib/travis/build/script/elm.rb

```diff
@@ -0,0 +1,183 @@
+module Travis
+  module Build
+    class Script
+      class Elm < NodeJs
+        # Default NodeJS version to install
+        DEFAULT_NODE_VERSION = '10.13.0'
+
+        DEFAULTS = {
+          elm: '0.19.0',
+          elm_test: '0.19.0',
+          elm_format: '0.19.0'
```

Comment:
The latest version is 0.8.1.

Reply:
In f53ea8a36a6ee1ab463e482bcd1bdd5e1f182708 I've changed these to use the `elm0.19.0` tag by default, which points to the appropriate version for the 0.19 release of Elm. (For `elm-format`, that's 0.8.1., so `npm install -g elm-format@elm0.19.0` gets you `elm-format` 0.8.1, and that tag will get updated if there are bugfixes on `elm-format` that are still compatible with Elm 0.19.)